nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
regarding user friendliness the issue we saw before providing a utility for more natural expression of duration is that people were very inconsistent in specifying the unit of time that a given value applied to and using at times periods which didn't make sense for the scenario.  This model allows the user to express in a much more natural way both the period of time and unit of time.